pkgs/development/python-modules/pygobject/default.nix

```diff
@@ -15,6 +15,12 @@ buildPythonPackage rec {
 
   patches = lib.optionals stdenv.isDarwin [
     ./pygobject-2.0-fix-darwin.patch
+  ] ++ lib.optionals (stdenv.isDarwin && stdenv.isAarch64) [
```

Comment:
Please make the patch unconditional

Reply:
Also without `lib.optionals stdenv.isDarwin`?

Reply:
Yes, otherwise it bitrots overtime because python-updates are mostly not tested to much on darwin.